config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -966,12 +966,6 @@ presubmits:
         - --test-cmd-args=--testconfig=testing/access-tokens/config.yaml
         - --test-cmd-args=--testoverrides=./testing/experiments/enable_prometheus_api_responsiveness.yaml
         - --test-cmd-args=--testoverrides=./testing/experiments/use_simple_latency_query.yaml
-        - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_configmaps.yaml
```

Comment:
We can only do that for master jobs, i.e. the ones checking out perf-tests repo from master branch, see line 943 (it uses release-1.16 where your changes are not present)

Reply:
Got it, thanks.